src/app_service/service/chat/service.nim

```diff
@@ -70,6 +76,7 @@ const SIGNAL_CHAT_HISTORY_CLEARED* = "chatHistoryCleared"
 const SIGNAL_CHAT_RENAMED* = "chatRenamed"
 const SIGNAL_CHAT_MEMBERS_ADDED* = "chatMemberAdded"
 const SIGNAL_CHAT_MEMBER_REMOVED* = "chatMemberRemoved"
+const SIGNAL_CHAT_MEMBER_UPDATED* = "chatMemberUpdated"
```

Comment:
@alaibe the issue when someone left a group chat is till here. If you create a chat and add members and if any of them left the chat members list (as well as members count) doesn't get updated. For that we have to monitor on `messages.new` signal and check `members` list of concrete chat received as part of the `chats` array and update the list.

Reply:
Yep, but this is not the issue I am fixing, I am fixing the issue of when you remove a member from a group he is not in the group list anymore. What you describe is the opposite, which I haven't test and is probably what you describe. Can we have a separate issue for that. This one is already fixing 3 issues